draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1479,28 +1483,29 @@ user agent MUST process the cookie as follows:
 
     2.  Set the cookie's domain to the canonicalized request-host.
 
-9.  If the cookie-attribute-list contains an attribute with an
+10. If the cookie-attribute-list contains an attribute with an
     attribute-name of "Path", set the cookie's path to attribute-value of
     the last attribute in the cookie-attribute-list with an attribute-name
```

Comment:
For clarity I recommend "the last attribute in the cookie-attribute-list with **both** an attribute-name..."

Reply:
Plot twist: _both_ reads as less clear to me (but is probably fine 🙈 )

Reply:
They sound about equally clear to me, but I went with "both". Thanks!

Reply:
Glad we covered all our bases then